homu/init.sls

```diff
@@ -36,9 +42,9 @@ homu:
   file.managed:
     - source: salt://{{ tpldir }}/files/cfg.toml
     - template: jinja
-    - user: servo
-    - group: servo
-    - mode: 644
+    - user: homu
+    - group: homu
+    - mode: 640
```

Comment:
Since this file is no longer world-readable, we'll need to run the tests with `sudo` from `.travis/dispatch.sh`. Please put this change in a separate commit first to make sure it doesn't break anything else.